tensorflow_model_optimization/g3doc/guide/clustering/clustering_with_keras.ipynb

```diff
@@ -0,0 +1,510 @@
+{
```

Comment:
<p>Let's go back to the comment where we discuss the headers and structure of the colab.</p><p> </p><p>I suggested there that we just do a single "persistence of accuracy" section. If the accuracy of the quantized, clustered TFLite model matches the TF one, then that seems to suggest that the clustered TFLite model would be good enough also.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>